src/wallet/rpcwallet.cpp

```diff
@@ -5058,6 +5080,7 @@ static const CRPCCommand commands[] =
     { "wallet",             "z_gettotalbalance",        &z_gettotalbalance,        false },
     { "wallet",             "z_mergetoaddress",         &z_mergetoaddress,         false },
     { "wallet",             "z_sendmany",               &z_sendmany,               false },
+    { "wallet",             "z_sendmanywithchangetosender", &z_sendmanywithchangetosender, false },

```

Comment:
This is **NOT** following the latest requirement that is suggested by @a-ok123 and @Dicklesworthstone 
We need to add _additional flag_ to `z_sendmany`, so the implementation must support:
    - `z_sendmany` (without additional flag): just behave old implementation. 
    - `z_sendmany -changetosendingaddress`: do the change utxo change back to the old address.
But you are creating a brand new command `z_sendmanywithchangetosender`.